packages/admin-stories/src/docs/components/ClearInputButton/ClearInputButton.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs/blocks";
+
+<Meta title="Docs/Components/ClearInputButton" />
+
+# ClearInputButton
+
+The ClearInputButton is a Button which is used inside a clearable Input Field. It can be used inside an InputAdornment of an InputField.
+The ClearInputButton extends the props from the [Material UI ButtonBase](https://mui.com/api/button-base/) component with the `icon` prop.
```

Comment:
mui.com is the documentation for MUI v5. The documentation for v4, which we are currently using, is located at https://v4.mui.com/api/button-base/#buttonbase-api. Should we care about this in our documentation? Imo it depends on how long we wait before upgrading to MUI v5.